source/renderer/app/api/ada/types.js

```diff
@@ -91,15 +91,15 @@ export type AdaWallet = {
     data: ?{
       estimatedCompletionTime: {
         quantity: number,
-        unit: number
+        unit: string
```

Comment:
@daniloprates 

I changed `unit` from a `number` to a `string` based on the docs.

<img width="644" alt="screen shot 2018-08-06 at 8 26 59 pm" src="https://user-images.githubusercontent.com/13347886/43762927-2c844980-99de-11e8-8e93-cb71e15bbb8c.png">